examples/function-values/function_values.md

```diff
@@ -1,7 +1,12 @@
 # Function values
 
-Functions are values and work as closures. Function type is a separate basic type. Anonymous function and type syntax look like function definition without the name.
+In Ballerina, a function is also a value, implying that it can be stored in variables, and passed to or returned from functions.  A function value can be executed by calling it. A value cannot be assigned to the function identifier.
```

Comment:
```suggestion
In Ballerina, a function is also a value implying that it can be stored in variables and passed to or returned from the functions.  A function value can be executed by calling it. A value cannot be assigned to the function identifier.
```